app/components/CVDisplay.tsx

```diff
@@ -0,0 +1,101 @@
+import { CVData } from '~/lib/schemas/classicCVTemplateSchema'
+
+type CVDisplayProps = {
+  data: CVData
+}
+
+export const CVDisplay: React.FC<CVDisplayProps> = ({ data }) => {
+  return (
+    <div className="cv-container space-y-8 p-4 border rounded-lg bg-white shadow-md">
+      {/* Basic Information */}
+      <section>
+        <h1 className="text-3xl font-bold">{data.name}</h1>
+        <h2 className="text-xl text-gray-700">{data.positionTitle}</h2>
+        {data.portfolio && (
+          <p className="text-sm text-blue-500">
+            <a href={data.portfolio} target="_blank" rel="noopener noreferrer">
+              Portfolio
+            </a>
+          </p>
+        )}
+      </section>
+
+      {/* Contact Information */}
+      <section>
+        <h3 className="text-lg font-semibold">Contact Information</h3>
+        <ul className="list-disc ml-6">
+          <li>Phone: {data.contact.phone}</li>
+          <li>Email: {data.contact.email}</li>
+          {data.contact.linkedin && <li>LinkedIn: {data.contact.linkedin}</li>}
+        </ul>
+      </section>
+
+      {/* Summary */}
+      <section>
+        <h3 className="text-lg font-semibold">Summary</h3>
+        <p>{data.summary}</p>
+      </section>
+
+      {/* Skills */}
+      <section>
+        <h3 className="text-lg font-semibold">Skills</h3>
+        <ul className="list-disc ml-6">
```

Comment:
**suggestion**: Consider using a more descriptive key for the list items, such as `skill.name`, if available, to avoid potential issues with list reordering.